### DISCUSSION TITLE
personal question (sorry, is this the place?): titlebar customization

### DISCUSSION THREAD

**Discussion by akirapink** [SebastJava/mint-themes#46] [Q&A] [answered]
i don't really like the titlebar theme that comes with mint-yz
i think it'd probably be kind of obnoxious of me to ask for this as a variant of the gtk theme
so how could i edit the existing mint-yz theme files myself to use the old mint-y-legacy title bar icons and spacing?

i'm terrribly sorry to ask here! this is probably not the right place but gtk themes are big and i. tried my best editing them but i don't know css as well as i think i do

(i think i'd rather just a snippet i can slap onto ~/.config/gtk-3.0/gtk.css either way though than a diff for the source, the gtk development kit is...too large for me to bother)

**Comment by SebastJava** [comment, marked as answer]
> Personal question (sorry, is this the place?)

No, i transferred this into https://github.com/SebastJava/mint-yz-theme/discussions/

> So how could i edit the existing Mint-Yz theme files myself to use the old `mint-y-legacy` title bar icons and spacing?

I would need more information about your system to answer this. Version 21? Cinnamon? I guess you are on LinuxMint 21. Otherwise you could easily choose a different "Window borders" theme in your Preferences > Themes or Appearance.

So, if you are on some LinuxMint 21, this theme modification sounds hard to do. I don't know how to change this. Before LM version 21, the window borders were defined in the metacity-1 directory. Now this directory is still there but it is not used anymore. AFAIK, the drawing is now defined directly into the `gtk.css` or `gtk-dark.css` file, somewhere in there. I don't know where in these files. And I don't know how you could convert the metacity-1/NAME.svg files from the `mint-y-legacy` into CSS coding...

> I'm terribly sorry to ask here!

Don't feel so bad. You are giving me an opportunity to develop my discussion skills. But this is not the best possible timing. I am currently very busy, overloaded. I suggest you open a new topic on https://forums.linuxmint.com/viewforum.php?f=211 for this question. And please, don't forget to mention which Mint DE and version you are using. Examples: LinuxMint Cinnamon 21 or LinuxMint Mate 20.3...

**Comment by fblais** [comment]
I can be wrong, but I think MATE still uses the older design where the metacity theme can be selected.
The OP could switch to MATE for this feature.
Switching to XFCE is another alternative. You can select the window decorations individually.
I understand this requires dropping Cinnamon if running Mint 21...
